### ISSUE TITLE
Relation to `bevy_web_asset`?

### ISSUE THREAD

**Issue by johanhelsing** [lizelive/bevy_http#1]
Hey, just saw this. Sounds like it will make my crate obsolete if it's merged into bevy?

Just wanted to check in so we don't do duplicate work.

https://github.com/johanhelsing/bevy_web_asset/pull/18

**Comment by lizelive** [comment]
yeah im working on that. http is really important and bevy already has support for http on webasm
